content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -1,17 +1,28 @@
 ---
-title: Optimization Improvements in DVC 1.0
+title: Remote Optimization Improvements in DVC 1.0
 date: 2020-06-29
 description: |
   An overview of how data synchronization to and from remote storage is optimized in DVC 1.0.
+picture: ''
 author: peter_rowlands
 ---
 
-One of the key features provided by DVC is the ability to efficiently sync
-versioned datasets between a user's local machine and
-[remote storage](https://dvc.org/doc/command-reference/remote), and version 1.0
-includes several performance optimizations related to synchronizing data with
-remotes. This post provides an overview of some of the ways in which remote
-performance has been optimized in DVC.
+One of the key features provided by DVC is the ability to efficiently
+synchronize versioned datasets between a user's local machine and
+[remote storage](https://dvc.org/doc/command-reference/remote), and our users
+have frequently shown the need for us to optimize this process wherever
+possible.
+
+![](/uploads/images/2020-06-29/optimization_screenshot.png '=600')
+
+DVC 1.0 includes several performance improvements with regard to how DVC
+synchronizes data to and from remotes. In particular, for this release we
+focused on optimizing performance in situations where users with must regularly
```

Comment:
I'd like to know a bit more about why the problem is hard. What are the main limits on transfer speed? When are they most felt by users? It sounds like users notice performance often when they are regularly using `dvc push` on a large dataset. 

*Also- let's give a range for datasets we're talking about. Do we mean big in terms of file count or disk space? A common question I get is if DVC works on really really big data (many TB) and probably want to give a clue that we're usually talking about MB-GB scales

Reply:
latest revision specifies that when discussing size we mean in terms of file count rather than bytes

Reply:
`where users with` - some typo here?